dispatches/models/fossil_case/thermal_oil/thermal_oil.py

```diff
@@ -165,7 +165,7 @@ def initialize(self, state_args={}, state_vars_fixed=False,
         else:
```

Comment:
Remove all of this `if/else`.

Reply:
hey @klfrick2 can you take a quick look at this please, I fixed one issue but it seems like there is more.